packages/fxa-auth-server/scripts/stripe-products-and-plans-to-firestore-documents.ts

```diff
@@ -2,18 +2,32 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 import program from 'commander';
+import path from 'path';
 import Container from 'typedi';
 
 import { setupProcessingTaskObjects } from '../lib/payments/processing-tasks-setup';
 import { AppConfig } from '../lib/types';
-import { StripeProductsAndPlansConverter } from './stripe-products-and-plans-to-firestore-documents/stripe-products-and-plans-converter';
+import {
+  StripeProductsAndPlansConverter,
+  OutputTarget,
+} from './stripe-products-and-plans-to-firestore-documents/stripe-products-and-plans-converter';
 
 const pckg = require('../package.json');
 
 const parseDryRun = (dryRun: boolean | string) => {
   return `${dryRun}`.toLowerCase() !== 'false';
 };
 
+const parseTarget = (target: any): OutputTarget => {
+  if (Object.values(OutputTarget).includes(target)) {
+    return target;
+  } else {
+    throw new Error('Invalid target option.');
+  }
+};
+
+const parseTargetPath = (targetDir: string) => path.join(targetDir);
```

Comment:
Not sure I understand this.  What is that joining?

Reply:
Changed this back to `path.normalize`. Just something small to cleanup the input a little if necessary.